frontend/react-neko-chat/src/styles.css

```
@@ -1943,7 +1943,47 @@ body.electron-chat-window.subtitle-web-host .compact-chat-surface-shell {
+  align-items: center;
+  justify-content: center;
+  cursor: grab;
+  touch-action: none;
+  color: #2f526b;
+}
+
+.compact-chat-input-drag-grip::before {
+  content: "";
+  width: 4px;
+  height: 18px;
+  background-image: radial-gradient(currentColor 0.9px, transparent 1.3px);
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)